mobile/src/common/enums/navigation/app-screen-name.enum.ts

```diff
@@ -11,10 +11,12 @@ enum AppScreenName {
   COURSE = 'Course',
   INTERVIEWS = 'Interviews',
   EDIT_COURSE_CATEGORY = 'Edit category',
+  EDIT_COURSE = 'Edit course',
```

Comment:
Isn't it same course as EDIT_COURSE_CATEGORY. If so, we need to reuse it
```suggestion
  EDIT_COURSE = 'Edit course',
```

Reply:
I thought in the future EDIT_COURSE will have additional logic, not just category editing

Reply:
Maybe the name EDIT_COURSE_CATEGORY confused me, and I decided that we edit only the category, and the whole course in another screen. I'll redo.